test;
[RemoveExceptions]
    class Test {
    void fo<caret>o() throws RuntimeException {
    }
  }
